packages/app-project/src/components/AuthModal/AuthModal.js

```diff
@@ -0,0 +1,61 @@
+import { Modal, Tabs, Tab, withThemeContext } from '@zooniverse/react-components'
+import { Box } from 'grommet'
+import { func, string } from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+import authModalTheme from './theme'
+import LoginForm from './components/LoginForm'
+import withOnlyRenderOnBrowser from './shared/components/withOnlyRenderOnBrowser'
+
+const StyledTabs = styled(Tabs)`
+  div[role="tabpanel"] {
+    background: ${props => props.theme.dark
+      ? props.theme.global.colors['dark-5']
+      : props.theme.global.colors['white']
+    };
+  }
+  button[role="tab"][aria-selected="true"] > div {
+    background: ${props => props.theme.dark
+      ? props.theme.global.colors['dark-5']
+      : props.theme.global.colors['white']
+    };
+  }
+`
+
+function AuthModal (props) {
+  const {
+    activeIndex,
+    className,
+    closeModal,
+    onActive,
+  } = props
+
+  return (
+    <Modal
+      active={activeIndex > -1}
+      className={className}
+      closeFn={closeModal}
+    >
+      <StyledTabs activeIndex={activeIndex} onActive={onActive}>
+        <Tab title='Sign In'>
+          <Box width='medium'>
+            <LoginForm closeModal={closeModal} />
+          </Box>
+        </Tab>
+        <Tab title='Register'>
+          <Box width='large'>
+            bar
+          </Box>
+        </Tab>
+      </StyledTabs>
+    </Modal>
+  )
+}
+
+AuthModal.propTypes = {
+  className: string,
+  closeModal: func.isRequired,
+}
+
+export default withOnlyRenderOnBrowser(withThemeContext(AuthModal, authModalTheme))
```

Comment:
We could wrap the `Modal` component directly with `withOnlyRenderOnBrowser` in the lib-react-components (and remove the check I had added that doesn't actually work). That's what I had done in the close PR where I was attempting updates on these... We'll probably always want this check until the underlying Layer becomes SSR friendly.